[16.0][ADD] pos_cash_opening_popup_no_reset (#1449)
OCA/pos: pos_cash_opening_popup_no_reset/readme/DESCRIPTION.rst

> On the cash opening popup removes the cash total being reset to 0 when clicking on the calculator..

Reviewer: ```suggestion
Disable reset of the total cash to 0 when clicking on the calculator on the cash opening popup.
```
